### ISSUE TITLE
Version 1.1.0: Module not found: Error: Can't resolve 'os'

### ISSUE THREAD

**Issue by Mehdi203** [jufab/opentelemetry-angular-interceptor#109]
Hi,
I just tried the latest version(1.1.0) but my app failed to compile. I didn't have this problem with the previous version 1.0.1 and wondering what's changed and how to resolve it. This is the error that I get:
```
`
 Error: Module not found: Error: Can't resolve 'os' in '/home/.../node_modules/@opentelemetry/resources/build/esm/detectors'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "os": require.resolve("os-browserify/browser") }'
        - install 'os-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "os": false }
`
```
p.s:
I tried the solution provided in this link but it didn't work:
[https://www.amadousall.com/angular-cli-module-not-found-error-cant-resolve/](https://www.amadousall.com/angular-cli-module-not-found-error-cant-resolve/)

**Comment by mehdi-qhr** [comment]
Just a quick update on the issue above. The opentelemetry team resolved the issue in version 1.3.1 (I was using version 1.3.0 for opentelemetry libraries). The above issue is listed as one of the enhancements in this link:
[opentelemetry-js](https://github.com/open-telemetry/opentelemetry-js/releases/tag/v1.3.0)
In summary, use version 1.3.1 instead of 1.3.0 to resolve this issue.

**Comment by Mehdi203** [comment]
Just a quick update on the issue above. The opentelemetry team resolved the issue in version 1.3.1 (I was using version 1.3.0 for opentelemetry libraries). The above issue is listed as one of the enhancements in this link:
[opentelemetry-js](https://github.com/open-telemetry/opentelemetry-js/releases/tag/v1.3.0)
In summary, use version 1.3.1 instead of 1.3.0 to resolve this issue.

**Comment by jufab** [comment]
I updated project with 1.3.1/0.29.2 => Now, I'm going to release it and close this issue
